README.md

```diff
@@ -57,11 +48,15 @@ const bynder = new Bynder({
   baseURL: "https://portal.getbynder.com/",
   clientId: "<your OAuth2 client id>",
   clientSecret: "<your OAuth2 client secret>",
-  redirectUri: "<url where user will be redirected after authenticating>"
+  redirectUri: "<url where user will be redirected after authenticating>",
+  // If you're using client credentials grant
+  // you also need to pass the following property
+  clientCredentials: true
```

Comment:
You don't want to provide a redirectUri if you're using Client Credentials grant, it's used exclusively for the Authorization Code grant.